src/Database.php

```diff
@@ -155,6 +158,67 @@ public function __toString()
         return $this->databaseName;
     }
 
+    /**
+     * Runs an aggregation framework pipeline on the database for pipeline
+     * stages that do not require an underlying collection, such as $currentOp
+     * and $listLocalSessions. Requires MongoDB >= 3.6
+     *
+     * Note: this method's return value depends on the MongoDB server version
+     * and the "useCursor" option. If "useCursor" is true, a Cursor will be
+     * returned; otherwise, an ArrayIterator is returned, which wraps the
+     * "result" array from the command response document.
```

Comment:
Per earlier discussions in the doc files, I think we can remove this or replace it with a note that the `useCursor` option should just be left alone.